CHANGELOG.md

```diff
@@ -13,10 +13,17 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - [#386](https://github.com/embedded-graphics/embedded-graphics/pull/386) Added the `delta` method to `Line` to compute the difference between start and end points.
 - [#450](https://github.com/embedded-graphics/embedded-graphics/pull/450) Added `ColorConverted` and `DrawTargetExt::color_converted` to support color conversion for draw targets.
 - [#438](https://github.com/embedded-graphics/embedded-graphics/pull/438) Added majority CSS web colors as associated `const`s to the RGB color types.
+- [#470](https://github.com/embedded-graphics/embedded-graphics/pull/470) Added support for external text renderers. External text renderers can be implemented using the new `TextStyle` trait.
 
 ### Changed
 
 - **(breaking)** [#466](https://github.com/embedded-graphics/embedded-graphics/pull/466) Upgrade Nalgebra from 0.19.0 to 0.23.0.
+- **(breaking)** [#470](https://github.com/embedded-graphics/embedded-graphics/pull/470) Renamed `Font`, `TextStyle` and `TextStyleBuilder` to `MonoFont`, `MonoTextStyle` and `MonoTextStyleBuilder`.
```

Comment:
So we can keep track of what needs to go in the migration guide instead of doing it all at once, could you add a section about these changes? Bullet points are fine if you prefer - I can flesh them out later.

Reply:
I've added some very basic bullet points to the migration guide. I'll try to remember to update the guide when I add more text features in future PRs.